di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/CoreStub.java

```diff
@@ -28,6 +29,10 @@ public CoreStub() throws Exception {
 
     private void initRoutes() throws Exception {
         CoreStubHandler coreStubHandler = new CoreStubHandler(new HandlerHelper(getEcPrivateKey()));
+        BasicAuthHandler basicAuthHandler = new BasicAuthHandler();
```

Comment:
If ENABLE_BASIC_AUTH's value is true. We can create this class. I think, so we don't create an unnecessary class for false config value.